app/uk/gov/hmrc/emcstfereferencedata/config/AppConfig.scala

```diff
@@ -26,4 +26,5 @@ class AppConfig @Inject() (val config: Configuration, servicesConfig: ServicesCo
   lazy val crdlCacheUrl: String  = servicesConfig.baseUrl("crdl-cache")
   lazy val crdlCachePath: String = config.get[String]("microservice.services.crdl-cache.path")
   lazy val importRefDataSchedule: String = config.get[String]("import-reference-data.schedule")
+  lazy val internalAuthToken: String = config.get[String]("internal-auth.token")
```

Comment:
This token is already present in app-config-{qa,staging,production} - the PlatOps team sets this up for us